Sources/SKTestSupport/IndexedSingleSwiftFileTestProject.swift

```diff
@@ -19,6 +19,8 @@ import SourceKitLSP
 import SwiftExtensions
 import TSCBasic
 import ToolchainRegistry
+import Basics
+import PackageModel
```

Comment:
I would prefer not to add a dependency on SwiftPM here. We do have compilation conditions to build SourceKit-LSP without SwiftPM and they would need to be passed through here as well. Could we instead copy the relevant parts of `WindowsPlatformInfo`?

Reply:
Sure